tf/start.sh

```diff
@@ -42,6 +42,10 @@ do
     # train all networks
     for netarch in ${NETARCHS[@]}
     do
+      echo "Saving .yaml changes:"
+      git add $CONFIGDIR/$netarch.yaml
+      git commit -m "Configuration change"
+      git push https://github.com/LeelaChessZero/lczero-training.git running_conf
```

Comment:
I think this is good indeed!
Probably the last command should be performed in parallel using `&`.

Reply:
Done